src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.file.FileReader;
+import core.basesyntax.file.FileReaderImpl;
+import core.basesyntax.file.FileWriter;
+import core.basesyntax.file.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.report.ReportGenerator;
+import core.basesyntax.report.ReportGeneratorImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.service.StorageService;
+import core.basesyntax.service.StorageServiceImpl;
+import core.basesyntax.service.operation.BalanceOperationHandler;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.PurchaseOperationHandler;
+import core.basesyntax.service.operation.ReturnOperationHandler;
+import core.basesyntax.service.operation.SupplyOperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        FileReader fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("src/main/resources/fruit_shop.csv");
```

Comment:
Avoid using the absolute path to a resource. Use the relative path 'src/main/resources/fruit_shop.csv' as a parameter for the read method. This ensures that the project is portable and can run on any machine without path issues.